docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -406,29 +406,18 @@ this person as a dependent.
 
 **For simulant under 18:**
 
-- Child is a biological or adopted child to reference person 
+- Child is a biological, adopted, foster or step child to reference person 
     * Assign reference person 
-- Child is foster child 
-    * Assign reference person 
-- Child is step child 
-    * If there is a spouse / partner of reference person assign them 
-    * Otherwise assign the reference person 
-- Child is grandchild of reference person 
-    * If there is a child (biological, adopted, foster) of reference person, assign them (if multiple, assign at random) 
-    * Otherwise assign reference person 
-- Child is brother/sister to reference person 
-    * If there is a parent of the reference person, assign them 
-    * Otherwise assign the reference person 
-- Child is other relative to reference person 
-    * Assign a relative of the reference person who is between 20 and 45 years older than the child. If there are multiple, select one of the same race/ethnicity. If there are multiple of the same race/ethnicity, assign at random. 
+- Child is any other relative to reference person (NOT roommate/housemate or other nonrelative)
+    * Assign a relative of the reference person who is between 20 and 45 years older than the child. If there are multiple, assign at random. 
```

Comment:
could you define "relative" here?

Reply:
i know this seems like it should be obvious, but the concept "relative" doesn't currently exist in the sim, so we need to pin it down exactly in order to implement it.

for example, i think you would probably mean any two people A and B who currently live in a non-GQ house together, such that person A and person B both have one of the following "relationship to reference person"s:

     "Reference person",
    "Opp-sex spouse",
    "Opp-sex partner",
    "Same-sex spouse",
    "Same-sex partner",
    "Biological child",
    "Adopted child",
    "Stepchild",
    "Sibling",
    "Parent",
    "Grandchild",
    "Parent-in-law",
    "Child-in-law",
    "Other relative",
    "Foster child"

as opposed to, say, that plus anyone who has a tracked mother or guardian who now lives in another house, or some other definition?

Reply:
actually -- i think i have a recollection of Zeb writing something that defines the concept "relative". im not sure if it's been merged yet (or if i made that up?) but if that's somewhere else in the docs, would you reference or link to that from here? that way this section on its own contains all the details we need in order to implement

Reply:
Sure, makes sense. I added some clarification on "relative", which is much easier to define in the negative (anything except XYZ), let me know if this makes sense. Since this is run on initialization only, I don't think we will encounter any tracked mothers living elsewhere?